rero_mef/agents/mef/api.py

```diff
@@ -183,6 +183,33 @@ def create_deleted(cls, record, dbcommit=False, reindex=False):
         }
         return cls.create(data=data, dbcommit=dbcommit, reindex=reindex)
 
+    @classmethod
+    def get_latest(cls, pid_type, pid):
+        """Get latest Mef record for pid_type and pid.
+
+        :param pid_type: pid type to use.
+        :param pid: pid to use..
+        :returns: latest record.
+        """
+        search = AgentMefSearch().filter({'term': {f'{pid_type}.pid': pid}})
+        if search.count() > 0:
+            data = next(search.scan()).to_dict()
+            new_pid = None
+            if relation_pid := data.get(pid_type, {}).get('relation_pid'):
```

Comment:
COSMETIC : As you filter the query on `{pid_type}.pid`, you can be sure that this key exists. You could use simpler `data[pid_type].get('relation_pid')`. But your solution works well too ;-)